src/app/admin/organization/table.tsx

```diff
@@ -35,11 +37,16 @@ export function MembersAdminTable() {
 
     return (
         <Flex direction={'column'}>
+            {globalError && (
+                <Alert icon={<Warning size={16} />} color="red" mb="md">
+                    {globalError}
```

Comment:
I think i see what you're trying to do here - i think we can avoid all this prop drilling (ie passing down `onError` to all the different forms, and instead just use [mantine's notifications](https://mantine.dev/x/notifications/). you can do something like:
```notifications.show({ message: 'Your error message', color: 'red' })```

wherever the error is actually coming out, ie [here](https://github.com/safeinsights/management-app/pull/88/files#diff-4135e16cee7d3541903c61727bec9d1aadc891f7879b0dcf46f8f51d92a07015R25)